packages/fxa-settings/src/components/PageDisplayName/index.test.tsx

```diff
@@ -6,15 +6,64 @@ import 'mutationobserver-shim';
 import React from 'react';
 import '@testing-library/jest-dom/extend-expect';
 import { act, fireEvent, screen } from '@testing-library/react';
-import PageDisplayName from '.';
+import PageDisplayName, { UPDATE_DISPLAY_NAME_MUTATION } from '.';
 import {
+  createCache,
   MockedCache,
   MOCK_ACCOUNT,
   renderWithRouter,
 } from 'fxa-settings/src/models/_mocks';
 import { AuthContext, createAuthClient } from 'fxa-settings/src/lib/auth';
+import { HomePath } from 'fxa-settings/src/constants';
+import { GraphQLError } from 'graphql';
+import { MockedProvider } from '@apollo/client/testing';
+import { Account, GET_ACCOUNT } from 'fxa-settings/src/models';
+import AlertBarContext, {
+  AlertBarRootAndContextProvider,
+} from 'fxa-settings/src/lib/AlertBarContext';
```

Comment:
Although I can see at least one example to the contrary, I think we're trying to be consistent in using relative paths for imports from within fxa-settings. Could we change to that here?

Reply:
yeah, it vscode likes to auto-import this way. i can't quite remember why we decided to prefer relative... something about the ts build step i think

Reply:
Yeah, a lot of my imports are just results of ctrl+.  I'll change it.